cms/core/models/base.py

```diff
@@ -132,6 +133,24 @@ def has_ons_embed(self) -> bool:
 
         return False
 
+    def publish(self, *args: Any, **kwargs: Any) -> Any:
```

Comment:
hmm... I wonder if this is enough. For example the page edit view doesn't call `page.publish()` directly

Reply:
Huh, so you're right: https://github.com/wagtail/wagtail/blob/47fc2008640ece76be97f3d03da224b903a99047/wagtail/admin/views/pages/edit.py#L593 :sob:

Reply:
After much pain, this is fixed in https://github.com/ONSdigital/dis-wagtail/pull/191/commits/2989218ffafbf29a62e8ee60b88220e3462b4b35